packages/cli/commands/upload.js

```diff
@@ -29,13 +28,20 @@ const {
 const { uploadPrompt } = require('../lib/prompts/uploadPrompt');
 const { validateMode, loadAndValidateOptions } = require('../lib/validation');
 const { trackCommandUsage } = require('../lib/usageTracking');
-const { getThemePreviewUrl } = require('@hubspot/cli-lib/lib/files');
+const {
+  getThemePreviewUrl,
+  getThemeJSONPath,
+} = require('@hubspot/cli-lib/lib/files');
 const { i18n } = require('@hubspot/cli-lib/lib/lang');
-
 const i18nKey = 'cli.commands.upload';
 const { EXIT_CODES } = require('../lib/enums/exitCodes');
+const {
+  FieldsJs,
+  isProcessableFieldsJs,
+  cleanupTmpDirSync,
+} = require('@hubspot/cli-lib/lib/handleFieldsJs');
 
-exports.command = 'upload [--src] [--dest]';
+exports.command = 'upload [--src] [--dest] [--option]';
```

Comment:
Same suggestion about mentioning option here. We only need to list options here that are required inputs for the command to successfully run.